bats/tests/common-setup.bash

```diff
@@ -60,7 +62,9 @@ _common_teardown_file() {
   echo " " >&3
   CLEAN_UP_TEST_ENVIRONMENT=true
   # print docker logs
-  #docker-compose -f "$DOCKER_COMPOSE_PATH" logs >&3
+  local docker_logs
+  docker_logs=$(docker-compose -f "$DOCKER_COMPOSE_PATH" logs)
+  log DEBUG "${docker_logs}"
```

Comment:
Agreed, I was thinking about something similar so I'm glad that you came with the same conclusion. The amount of the docker logs can be overwhelming so it would help. Will add it.

Reply:
added